release/authorization/README.md

```diff
@@ -0,0 +1,419 @@
+}
+```
+
+permits the singular user "stevie" to access the path:
+
+```shell
+    /this/is/a/message_path
+```
+
+Additionally, "stevie" is permitted access to all paths below the defined path,
+in a READ only mode, such as:
+
+```shell
+    /this/is/a/message_path/the
+    /this/is/a/message_path/the/one
+    /this/is/a/message_path/the/one/that
+    /this/is/a/message_path/the/one/that/knocks
+```
+
+The second policy rule:
+
+```proto
+# Action memberss of family-group to run /this/is/a/different/message_path
+policy {
+  id: "two"
+  log_level: LOG_BRIEF
+  path {
+    origin: "foo"
+    elem { name: "this" }
+    elem { name: "is" }
+    elem { name: "a" }
+    elem { name: "different" }
+    elem { name: "message_path" }
+  }
+  action: PERMIT
+  level: READ
+  level: WRITE
+  group { name: "family-group" }
+}
+```
+
+example policy permits members or the family-group access to a single path, for
+reading or writing:
+
+```shell
+    /this/is/a/different/message_path
+```
+
+and all path elements beyond "message_path":
+
+```shell
+    /this/is/a/different/message_path/foo
+    /this/is/a/different/message_path/bar
+    /this/is/a/different/message_path/foo/baz/bing/boop
+```
+
+The third policy rule:
+
+```proto
+# Demonstrate a key with an attribute defined.
+policy {
+  id: "key"
+  log_level: LOG_BRIEF
+  path {
+    origin: "foo"
+    elem { name: "this" }
+    elem { name: "is" }
+    elem { name: "a" }
+    elem {
+      name: "keyed"
+      key{
+        key: "name"
+        value: "Ethernet1/2/3"
+      }
+    }
+    elem { name: "message_path" }
+  }
+  action: PERMIT
+  level: READ
+  group { name: "test-group" }
+}
+```
+
+Permits access by the "test-group" users to the keyed path, in a
+read only manner:
+
+```shell
+    /this/is/a/keyed[name=Ethernet1/2/3]/message_path
+```
+
+and all path elementas as beyond "message_path". The final policy rule:
+
+```proto
+# Demonstrate a key with a wildcard attribute.
+policy {
+  id: "wyld"
+  log_level: LOG_BRIEF
+  path {
+    origin: "foo"
+    elem { name: "this" }
+    elem { name: "is" }
+    elem { name: "a" }
+    elem {
+      name: "keyed"
+      key{
+        key: "name"
+        value: "*"
+      }
+    }
+    elem { name: "message_path" }
+  }
+  action: PERMIT
+  level: READ
+  group { name: "family-group" }
+}
+```
+
+permits access by the "family-group" users to the keyed path, with no
+restrictions on the key values, but still as read-only:
+
+```shell
+    /this/is/a/keyed[name=Ethernet1/2/3]/message_path
+    /this/is/a/keyed[name=POS3]/message_path
+    /this/is/a/keyed[name=Serial4/1]/message_path
+    /this/is/a/keyed[name=HSSI2]/message_path
+```
+
+Additionally, the path elements beyond "message_path" are available for access
+to this group as well.
+
+The wildcard character "*" (asterisk) is only able to be used as a value in
+keyed elements. The wildcard is only used to mask out all possible values but
+not portions of values, for instance:
```

Comment:
Is it worth specifying whether keys can be omitted? In some cases, I think we allow `/this/is/a/keyed/message_path` to mean the same as `/this/is/a/keyed[name=*]/message_path`, I need to check whether the reference documentation included this case as I didn't immediately see it in https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-path-conventions.md

Reply:
This sounds like an implementation detail.. sort of :)

how about:
  "...in keyed elements, If the key is missing in a path a wildcard is assumed. The wildcard is only...."